commands/python/command/restart.py

```diff
@@ -16,9 +22,9 @@ def parse_args(sub_parser):
 
 
 def restart_args(args):
-    restart(args.namespace, args.with_mount)
+    restart(args.namespace, args.charts, args.with_mount)
 
 
-def restart(namespace, with_mount):
+def restart(namespace, charts, with_mount):
     stop.stop(namespace)
-    start.start(namespace, False, with_mount)
+    start.start(namespace, charts, False, with_mount)
```

Comment:
stop everything in the namespace or just the charts (if not None)? 
Asking because help says 'Only start these charts..' does it imply we stop only these charts

Reply:
good call. let me take a pass at that